src/commands/new.ts

```diff
@@ -242,6 +249,46 @@ export default {
     }
     // #endregion
 
+    // #region Prompt for Workflow type
+    const defaultWorkflow = "expo"
+    let workflow = useDefault(options.workflow) ? defaultWorkflow : options.workflow
+
+    if (workflow === undefined) {
+      const workflowResponse = await prompt.ask<{ workflow: Workflow }>(() => ({
+        type: "select",
+        name: "workflow",
+        message: "Choose a workflow:",
+        choices: [
+          {
+            // TODO: if we use name for nice cli display, it gets used as value? gluegun issue?
```

Comment:
@joshuayoes I was looking to use `name` for a nice display in the terminal but the underlying `value` as matching up with the `workflow` type.

But when I did this, I only get the `name` value back - tried using `result` to map the proper value but no luck. 

Do you have any experience here with gluegun/enquirer? Was reading [here](https://github.com/enquirer/enquirer#multiselect-prompt) even though we aren't going for multi select, but the map seemed a little magical

Reply:
Fixed in a pairing session with @joshuayoes